tests/integration/api/test_project.py

```diff
@@ -586,29 +586,32 @@ def test_partial_update_project(
     new_project: models.Project,
     default_scm_credential: models.EdaCredential,
     default_gpg_credential: models.EdaCredential,
-    admin_client: APIClient,
+    superuser_client: APIClient,
 ):
     assert new_project.eda_credential_id is None
     assert new_project.signature_validation_credential_id is None
     assert new_project.verify_ssl is False
 
+    new_org = models.Organization.objects.create(name="new org")
     new_data = {
         "name": "new-project-updated",
+        "organization_id": new_org.id,
         "eda_credential_id": default_scm_credential.id,
         "signature_validation_credential_id": default_gpg_credential.id,
         "scm_branch": "main",
         "scm_refspec": "ref1",
         "verify_ssl": True,
         "proxy": "http://user:$encrypted$@myproxy.com",
     }
-    response = admin_client.patch(
+    response = superuser_client.patch(
```

Comment:
@bzwei why did we have to switch to superuser_client, wouldn't admin_client have sufficed. Are we short circuiting the RBAC check

Reply:
pytest admin_client is created with roles in default org only.
This test is mainly focus on bringing back the missing field, not for rbac check.